src/dropdown/dropdown.component.ts

```diff
@@ -93,6 +94,10 @@ import { scrollableParentsObservable, isVisibleInContainer } from "./../utils/sc
 				[ngTemplateOutletContext]="getRenderTemplateContext()"
 				[ngTemplateOutlet]="displayValue">
 			</ng-template>
+			<ibm-icon-warning-filled16
```

Comment:
Can we use the directive form of the icon instead? I'm noticing some weird alignment issues ... so it'd be 

```html
<svg ibmIconWarningFilled16
    *ngIf="invalid"
    class="bx--dropdown__invalid-icon">
</svg>